src/rubric_eval/main.py

```diff
@@ -0,0 +1,217 @@
+"""
+How to run:
+
+1. git clone https://github.com/stella-z3g/RubricEval -b public_release
+2. cd RubricEval/ && pip install -e .
+3. python scripts/convert_dataset_to_rubriceval_input_format.py  # This will generate an example instructions.json
+4. rubric_eval get_rubrics --input_path=instructions.json --output_path=instructions_with_rubrics.json
+5. rubric_eval get_completions --model_configs="['gpt-4o-2024-05-13']" --input_path=instructions_with_rubrics.json --output_path_prefix=output_
```

Comment:
I would not use the prefix here but just the file path. We can do that by default it's just input path with prefix though.

Reply:
also why are the model configs a string of list of string? 
I think we should stick to alpaca_eval and just have a single model_config (i.e. just a string)

Reply:
sounds great - if model config is only a single string, we don't need the prefix